modules/standard/IO.chpl

```diff
@@ -7014,8 +7102,145 @@ iter fileReader.lines(stripNewline = false) {
 
   // Set the iostyle back to original state
   this._set_styleInternal(saved_style);
+}
 
-  this.unlock();
+// single locale parallel 'lines'
+@chpldoc.nodoc
+iter fileReader.lines(
+  param tag: iterKind,
+  stripNewline = false,
+  type t = string
+): t
+  where tag == iterKind.standalone && (t == string || t == bytes)
+{
+  on this._home {
+    try! this.lock();
+
+    const f = chpl_fileFromReaderOrWriter(this),
```

Comment:
for the PR message: the single-locale parallel iterator will also re-open the file